cli/spec/spec_helper.rb

```diff
@@ -22,6 +22,7 @@
     allow(Dir).to receive(:home).and_return('/tmp/')
   end
 end
+RSpec::Expectations.configuration.warn_about_potential_false_positives = false
```

Comment:
Oh that was left there. The reason was that recent version of rspec displays that warning on containers_command_spec
```
WARNING: Using `expect { }.not_to raise_error(SpecificErrorClass)` risks false positives, since literally any other error would cause the expectation to pass, including those raised by Ruby (e.g. NoMethodError, NameError and ArgumentError), meaning the code you are intending to test may not even get reached. Instead consider using `expect {}.not_to raise_error`. This message can be supressed by setting: `RSpec::Expectations.configuration.warn_about_potential_false_positives = false`.
```
Maybe the right solution is to change that spec file and remove error class from `raise_error` call?